OpenRA.Game/Network/Order.cs

```diff
@@ -231,9 +231,9 @@ public static Order Command(string text)
 			return new Order("Command", null, false) { IsImmediate = true, TargetString = text };
 		}
 
-		public static Order StartProduction(Actor subject, string item, int count)
+		public static Order StartProduction(Actor subject, string item, int count, bool queued)
```

Comment:
IMO it would be best if we default `queued = true`, then the changes to HackyAI and the Lua API can be dropped.

Reply:
Implemented.